xcodeproject/xcodeproj/target.go

```diff
@@ -107,22 +74,11 @@ func (t Target) IsUITestProduct() bool {
 	return filepath.Ext(t.ProductType) == ".ui-testing"
 }
 
-func (t Target) isAppClipProduct() bool {
+// IsAppClipProduct ...
+func (t Target) IsAppClipProduct() bool {
 	return t.ProductType == appClipProductType
 }
 
-// CanExportAppClip ...
-func (t Target) CanExportAppClip() bool {
```

Comment:
Its only usage is in `bitrise-init`, so I'm rather deleting this and making `IsAppClipProduct()` public. The code in `bitrise-init` already has access to the project, so it can call `Xcodeproj.DependentTargetsOfTarget()` and iterate over the results.

Reply:
Fair point! If the main concern is testability, is it enough if I extract this logic into a testable private method in `bitrise-init`, or you'd rather leave this method in `go-xcode`?

Reply:
A consideration was to make it harder to misuse this API (like using IsAppClipProduct instead of CanExportAppClip).
But as there is one use I am ok with removing it.